tests/manage/monitoring/conftest.py

```diff
@@ -140,8 +146,9 @@ def stop_mon():
         for mon in mons_to_stop:
             logger.info(f"Upscaling deployment {mon} back to 1")
             oc.exec_oc_cmd(f"scale --replicas=1 deployment/{mon}")
-        msg = f"Downscaled monitors {mons_to_stop} were not replaced"
-        assert check_old_mons_deleted, msg
+        if not split_index == 1:
```

Comment:
When would split_index be equal to 1?

Reply:
Especially when we need mon quorum to be lost, i.e 2 mon down out of 3 mons.
The operator cannot do anything with ceph or starting a new mon since the quorum is down so there is no chance of replacing old mons. Hence skipping it